llvm/lib/MC/MCELFStreamer.cpp

```diff
@@ -783,6 +786,65 @@ void MCELFStreamer::createAttributesSection(
   AttrsVec.clear();
 }
 
+void MCELFStreamer::createAttributesSection(
```

Comment:
If I've understood correctly, AArch64 uses a slightly different format to AArch32, RISC-V and Hexagon, so the function names should probably make this clear, instead of using overloading to select between the two formats.

Reply:
Does it has to be the functions names, or could it be explained in comments?